regnety/tuner.py

```diff
@@ -0,0 +1,110 @@
+from regnety.regnety.config.config import (
```

Comment:
I think it's okay to keep this script for reference here. However, I'd also make a brief note here about its usage i.e. we tuned hyperparameters using ImageNette to get a rough sense of the best set of hyperparameters to start our experiments with.

Reply:
>  brief note here about its usage

Do we want to note this at the beginning of the module or the docs?

Reply:
Noting this at the beginning of the script sounds good to me.